clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -2255,7 +2255,7 @@ protected DelegateConsumerRecordResult delegateConsumerRecord(
         versionedDIVStats.recordLeaderDIVCompletionTime(
             storeName,
             versionNumber,
-            LatencyUtils.getElapsedTimeInMs(beforeProcessingPerRecordTimestampNs),
+            LatencyUtils.getLatencyInMS(beforeProcessingPerRecordTimestampNs),
```

Comment:
These two function names are too confusing... We should think of a more clear name... and rename them.

Reply:
i am going to rename `getLatencyInMS` to `getElapsedTimeFromNSToMS` since it takes nano-seconds as the input and `getElapsedTimeInMs` to `getElapsedTimeFromMsToMs` as it takes milli-seconds as the input. Both of methods have output in MS.

Reply:
Ok SG, thanks.